apps/common-app/src/examples/RuntimeTests/tests/layoutAnimations/keyframe/basic.test.tsx

```diff
@@ -0,0 +1,193 @@
+    15: { transform: [{ scale: 1 }], easing: Easing.poly(1) },
+    30: { transform: [{ scale: 2 }], easing: Easing.poly(2), backgroundColor: 'gold' },
+    45: { transform: [{ scale: 1 }], easing: Easing.poly(3) },
+    60: { transform: [{ scale: 2 }], backgroundColor: 'blue', easing: Easing.poly(10) },
+    75: { transform: [{ scale: 1 }], easing: Easing.poly(4) },
+    85: { transform: [{ scale: 2 }], backgroundColor: 'purple', easing: Easing.poly(5) },
+    100: { transform: [{ scale: 1 }], backgroundColor: 'red', easing: Easing.poly(6) },
+  }).duration(1000);
+
+  const rotateAroundCornerAnimation = new Keyframe({
+    0: {
+      transform: [
+        { translateX: 50 },
+        { translateY: 50 },
+        { rotateZ: '0deg' },
+        { translateX: -50 },
+        { translateY: -50 },
+      ],
+    },
+    100: {
+      transform: [
+        { translateX: 50 },
+        { translateY: 50 },
+        { rotateZ: '360deg' },
+        { translateX: -50 },
+        { translateY: -50 },
+      ],
+    },
+  }).duration(1000);
+
+  const changeBordersAnimation = new Keyframe({
+    0: {
+      borderRadius: 20,
+      borderWidth: 5,
+      backgroundColor: '#3cb371ff',
+      borderColor: '#2f4f4fff',
+    },
+    25: {
+      borderRadius: 50,
+      borderWidth: 10,
+      borderColor: '#2f4f4fff',
+    },
+    50: {
+      borderRadius: 45,
+      borderWidth: 50,
+      borderColor: '#2f4f4fff',
+      backgroundColor: '#3cb371ff',
+    },
+    51: {
+      borderRadius: 45,
+      borderWidth: 50,
+      borderColor: '#2f4f4fff',
+      backgroundColor: '#ffd700ff',
+    },
+    100: {
+      borderRadius: 5,
+      borderWidth: 0,
+    },
+  }).duration(2000);
+
+  const elasticAnimation = new Keyframe({
+    0: { transform: [{ translateY: 0 }] },
+    100: { transform: [{ translateY: 200 }], easing: Easing.elastic(10) },
+  }).duration(2000);
+
+  const linearAnimation1 = new Keyframe({
+    0: {
+      transform: [{ translateY: 0 }],
+      easing: Easing.elastic(10), // This easing is unused
+    },
+    100: { transform: [{ translateY: 200 }] },
+  }).duration(500);
+
+  const linearAnimation2 = new Keyframe({
+    0: { transform: [{ translateY: 0 }] },
+    100: { transform: [{ translateY: 200 }] },
+  }).duration(500);
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const changeBordersAnimationBugRepro = new Keyframe({
+    0: {
+      borderRadius: 20,
+      borderWidth: 5,
+      backgroundColor: '#3cb371ff',
+      borderColor: '#2f4f4fff',
+    },
+    25: {
+      borderRadius: 50,
+      borderWidth: 10,
+      borderColor: '#2f4f4fff',
+    },
+    50: {
+      borderRadius: 50,
+      borderWidth: 50,
+      backgroundColor: '#3cb371ff',
+    },
+    51: {
+      borderRadius: 45,
+      borderWidth: 50,
+      borderColor: '#2f4f4fff',
+      backgroundColor: '#ffd700ff',
+    },
+    100: {
+      borderRadius: 5,
+      borderWidth: 0,
+    },
+  }).duration(5000);
```

Comment:
What's the deal with this?

Reply:
This animation reproduces a bug occurring on new architecture. One of the keyframes is skipped. The buggy animation is very similar to `changeBordersAnimation` defined above, however the second one works fine.

Reply:
Then please add some comment about it, maybe even a `TODO` comment.

Reply:
I've added
> // This is a reproduction of a weird bug happening on new architecture only.

GitHub doesn't see it and doesn't mark this as outdated, since the comment is in line 118 and you've marked lines 119-147.

Reply:
Thanks!